src/SuperVault.sol

```diff
@@ -657,7 +760,7 @@ contract SuperVault is BaseStrategy, ISuperVault {
         data.liqRequests = new LiqRequest[](_numberOfSuperforms_);
         bytes[] memory dataToEncode = new bytes[](_numberOfSuperforms_);
 
-        for (uint256 i; i < _numberOfSuperforms_; ++i) {
+        for (uint256 i; i < _numberOfSuperforms_;) {
```

Comment:
put `routerPlus` in an immutable variable?

Reply:
We are leaving it like this because there is an open possibility that a new version for it is released later (with the same interface) and because it's address can be modified in SuperRegistry (and it is a wrapper contract to core contracts)